specs/src/specs/consensus.md

```diff
@@ -15,27 +15,28 @@
+| `NAMESPACE_ID_SIZE`                     | `int`    | `28`         | `byte`  | Size of namespace ID in bytes.                                                                                                                                           |
+| `NAMESPACE_SIZE`                        | `int`    | `29`         | `byte`  | Size of namespace in bytes.                                                                                                                                              |
+| `NAMESPACE_ID_MAX_RESERVED`             | `uint64` | `255`        |         | Value of maximum reserved namespace (inclusive). 1 byte worth of IDs.                                                                                                    |
+| `SEQUENCE_BYTES`                        | `uint64` | `4`          | `byte`  | The number of bytes used to store the sequence length in the first share of a sequence                                                                                   |
+| `SHARE_INFO_BYTES`                      | `uint64` | `1`          | `byte`  | The number of bytes used for [share](data_structures.md#share) information                                                                                               |
+| `SHARE_RESERVED_BYTES`                  | `uint64` | `4`          | `byte`  | The number of bytes used to store the index of the first transaction in a transaction share. Must be able to represent any integer up to and including `SHARE_SIZE - 1`. |
+| `SHARE_SIZE`                            | `uint64` | `512`        | `byte`  | Size of transaction and blob [shares](data_structures.md#share), in bytes.                                                                                               |
+| `STATE_SUBTREE_RESERVED_BYTES`          | `uint64` | `1`          | `byte`  | Number of bytes reserved to identify state subtrees.                                                                                                                     |
+| `UNBONDING_DURATION`                    | `uint32` |              | `block` | Duration, in blocks, for unbonding a validator or delegation.                                                                                                            |
+| `v1.Version`                            | `uint64` | `1`          |         | First version of the application. Breaking changes (hard forks) must update this parameter.                                                                              |
+| `v2.Version`                            | `uint64` | `2`          |         | Second version of the application. Breaking changes (hard forks) must update this parameter.                                                                             |
```

Comment:
The only change here is I got rid of `VERSION_APP` and added v1.Version and v2.Version